rust/Dockerfile

```diff
@@ -1,20 +1,6 @@
-# Global args to use in build commands
 ARG ALPINE_VERSION="3.19"
-ARG CARGO_CHEF_VERSION="0.1.62"
-ARG RUSTUP_VERSION="1.26.0"
-ARG RUSTUP_x86_DOWNLOAD_SHA256="7aa9e2a380a9958fc1fc426a3323209b2c86181c6816640979580f62ff7d48d4"
-ARG RUSTUP_aarch64_DOWNLOAD_SHA256="b1962dfc18e1fd47d01341e6897cace67cddfabf547ef394e8883939bd6e002e"
-ARG RUST_VERSION="1.74.1"
 
-FROM alpine:${ALPINE_VERSION} as rust
-
-# Important!  Update this no-op ENV variable when this Dockerfile
-# is updated with the current date. It will force refresh of all
-# of the base images and things like `apk add` won't be using
-# old cached versions when the Dockerfile is built.
-ENV REFRESHED_AT=2024-02-22 \
-  LANG=C.UTF-8 \
-  TERM=xterm
+FROM rust:1.74-alpine as rust
```

Comment:
This makes the code shorter but does not make the CI faster. I'm not sure what it breaks.